### PR TITLE
CIV-8881 - CUI to CCD translation for evidence upload.

### PR DESCRIPTION
**Pull request by m-meulendijks-v1**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-8881

### Change description ###
Translate evidence upload information  from CUI to CCD.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by kurt-uk — DISMISSED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

File: src/test/unit/services/translation/caseProgression/convertToCCDEvidenceUpload.test.ts [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+  it('should convert CaseProgression to CCDClaim', () => {
+
+    const ccdClaim= {} as  CCDClaim;
+    const expectedOutput: CCDClaim = createCCDClaim();
+    const cuiClaim = createCUIClaim();
+
+    const actualOutput = toCCDEvidenceUpload(cuiClaim, ccdClaim);
+    expect(actualOutput).toEqual(expectedOutput);
+  });
+
```
Comment (pats-john) [inline src/test/unit/services/translation/caseProgression/convertToCCDEvidenceUpload.test.ts:99]: I recommend adding a test for a claim with only some of the document details and thrashing out if empty sections need to be considered for this or not.

**Review by pats-john — CHANGES_REQUESTED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

File: src/test/unit/services/translation/caseProgression/convertToCCDEvidenceUpload.test.ts (hunk 1 above)
Comment (m-meulendijks-v1) [reply src/test/unit/services/translation/caseProgression/convertToCCDEvidenceUpload.test.ts:99]: @pats-john I have added another tests that tests whether when ccd lists are only filled partially or with multiple elements.
I have not added a test for what happens when certain document fields are missing, because I think we'll need to change the way the pre-existing Document object is currently implemented to make certain fields mandatory.

**Review by kurt-uk — APPROVED** [review]

**Review by pats-john — APPROVED** [review]
